com.abisource.AbiWord.json

```diff
@@ -0,0 +1,218 @@
+{
+    "app-id" : "com.abisource.AbiWord",
+    "branch" : "3.0",
```

Comment:
everything on flathub must be on the 'stable' branch.

Reply:
done